library/coverage/result_spec.rb

```diff
@@ -16,6 +17,7 @@
     $LOADED_FEATURES.delete(@class_file)
     $LOADED_FEATURES.delete(@config_file)
     $LOADED_FEATURES.delete(@eval_code_file)
+    $LOADED_FEATURES.delete(@file)
```

Comment:
I suppose it should use `@with_begin_file` instead of `@file`.

Reply:
Yes, my mistake😅